pravega/pravega
# Issue 4569: Key Value Tables

**Change log description**  
Key-Value Tables added as a new top-level primitive. 
See https://github.com/pravega/pravega/wiki/PDP-39-Key-Value-Tables for details.

**Purpose of the change**  
Fixes #4569.

**What the code does**  
Merge feature branch `feature-key-value-tables` into branch `master`.

**How to verify it**  
Build must pass. System tests must pass.
Verify commits into `feature-key-value-tables` that they are made up of approved Pull Requests.
